packages/app/src/components/inspector/ColorPickerGradient.tsx

```diff
@@ -0,0 +1,197 @@
+import type Sketch from '@sketch-hq/sketch-file-format-ts';
+import { GridIcon, RowsIcon } from '@radix-ui/react-icons';
+import {
+  Spacer,
+  RadioGroup,
+  Select,
+  ListView,
+  Button,
+  Divider,
+  getGradientBackground,
+} from 'noya-designsystem';
+import { memo, useState } from 'react';
+import styled from 'styled-components';
+
+const PaddedSection = styled.section({
+  padding: '8px 10px',
+  display: 'flex',
+  flexDirection: 'column',
+});
+
+const Square = styled.div<{ color: string; selected?: boolean }>(
+  ({ theme, color, selected = false }) => ({
+    height: '25px',
+    width: '25px',
+    background: color,
+    border: `2px solid ${
+      selected ? 'rgb(132,63,255)' : theme.colors.popover.background
+    } `,
+    borderRadius: '4px',
+    cursor: 'pointer',
+  }),
+);
+
+const GridSmall = styled.div({
```

Comment:
Can we share most or all of these components with the Swatches grid?

Reply:
Do I keep this Compoments in SwatchesGrid ?

Reply:
Maybe move to a new file `inspector/ColorPickerAssetGrid`